packages/hopli/src/environment_config.rs

```diff
@@ -0,0 +1,173 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+/// Type of environment that HOPR node is running in
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq)]
+#[serde(rename_all(deserialize = "lowercase"))]
+pub enum EnvironmentType {
+    /// Production environment, on Gnosis chain
+    Production,
+    /// Staging environment, on Gnosis chain
+    Staging,
+    /// Development environment, on Gnosis chain
+    Development,
+}
+
+impl ToString for EnvironmentType {
+    /// convert environment type to string
+    fn to_string(&self) -> String {
+        match self {
+            Self::Production => "production".into(),
+            Self::Staging => "staging".into(),
+            Self::Development => "development".into(),
+        }
+    }
+}
```

Comment:
You could also implement the `Display` for Environment type and use that to generate the ToString.

Reply:
Thanks! Updated!